codalab/lib/worksheet_util.py

```diff
@@ -213,18 +213,23 @@ def get_formatted_metadata(cls, metadata, raw=False, show_hidden=False):
     return result
 
 
-def get_editable_metadata_fields(cls):
+def get_editable_metadata_fields(cls, state):
```

Comment:
Type annotate

Reply:
Can you confirm what you mean by this plz? I thought this meant adding param docs like `:param cls:` and `:param state:`

Do you mean that I should set a default like `state = ''`?